src/hardforks/constantinople.json

```diff
@@ -34,6 +34,10 @@
     "netSstoreResetClearRefund": {
       "v": 19800,
       "d": "Once per SSTORE operation for resetting to the original zero value"
+    },
+    "extcodehash": {
+      "v": 400,
+      "d": "Cost of EXTCODEHASH operation"
     }
```

Comment:
When I added the repricing for `EXTCODEHASH` in istanbul, I realized it didn't have the initial price set.

Reply:
Ah, just realizing that we had this [discussion](https://github.com/ethereumjs/ethereumjs-vm/pull/581/files/56350e1f22536207e3de7da3a14f83871afd4005#diff-358e6857042c422cdc988b1104649934) already on the VM side (sorry, didn't read closely your second point on the TODO list), and we found a solution here already (I stumbled upon the exact same thing).

Can you please remove the last two price-related commits and just leave this to the Istanbul block number update?

Side note: sorry, #61 should have been closed. Please don't trust issues being up-to-date too much atm, this is one larger part of my back log re-taking on a more active issue management. In doubt ask before starting on something, otherwise this might lead to frustration and/or double work.